website/content/docs/commands/print.mdx

```diff
@@ -0,0 +1,14 @@
+---
+layout: docs
+page_title: version - Command
+description: The "print" command prints the Vault token currently in use.
+---
+
+# print
+
+The `print` command prints the Vault token currently in use. The only available subcommand is `token`.
+
+```shell-session
+$ vault print token
+hvs.XXXXXXXXXXXXXXXXXXXXXXXX
+```
```

Comment:
I would suggest add **Example** header for style consistency.

````suggestion
## Examples

Print the current token.

```shell-session
$ vault print token
hvs.CAESICaie3Dm0_Hx001QuMabo1IXnyKkx_FuE14MH7zir_bqGh4KHGh2cy5wQnJsZzZ6WG82b29HUlI3eFdEQ0NPQzQ
```
````